101-maps-create/azuredeploy.parameters.json

```diff
@@ -0,0 +1,12 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "account_name": {
```

Comment:
remove these since you have defaultValues in the template

Reply:
Remove the parameters file entirely?

Reply:
no, just empty the parameters property, e.g. 
```json
{
  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
  "contentVersion": "1.0.0.0",
  "parameters": { }
}
```